CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGE LOG
 ## 4.1.1 (UPCOMING)
 
 * Fixed incorrect floating IP IDs
+* Fixed missing firewall client method
```

Comment:
Can you please rebase against the 4.2 branch, and add this entry to the 4.2.0. We follow semver, and this PR is not a fix, but new features. :)